decam/SConscript

```diff
@@ -34,3 +35,12 @@ linearizerOutDir = "linearizer"
 command = "%s decam/makeLinearizer.py $SOURCE --force" % (python,)
 
 makeLinearizer = env.Command(linearizerOutDir, linearizerInFile, command)
+
+
+# Ingest defects
+pipe_tasks_dir = getPackageDir('pipe_tasks')
+data_dir = getPackageDir('obs_decam_data')
+command = (f"{pipe_tasks_dir}/bin/ingestDefects.py decam/CALIB/ {data_dir}/decam/defects "+
+            "--calib decam/CALIB --config clobber=True")
```

Comment:
It looks like this means defects will automatically be ingested into the already-setup obs_decam_data repository each time obs_decam is built. That's nice! How should a user take advantage of this when they are working to ingest all the necessary calibs to run processCcd on some decam data? Please document this somewhere.

Reply:
This will be documented in the forth coming community post.  Unfortunately, I don't think this will do all you want/need.  It will ingest the defects into a calib repository in `$OBS_DECAM_DIR/decam/CALIB`.  That will be a fully working calib repository, but without the other calibs you may want.  Since the gen2 system cannot handle two calib roots, you will either need to ingest your flats, biases, etc into the calib repository produced at scons time, or ingest the defects into the calib repository you are using to run processCcd.